pki/util.go

```diff
@@ -135,6 +135,34 @@ func GenerateCACertAndKey(commonName string, privateKey *rsa.PrivateKey) (*x509.
 	return kubeCACert, rootKey, nil
 }
 
+func GetIPHostAltnamesForHost(host *hosts.Host) *cert.AltNames {
+	ips := []net.IP{}
```

Comment:
nit: to avoid an unnecessary allocation, it's better to declare the slice in a form of `var ips []net.IP`. As a nil slice is functionally equivalent to a zero-length slice